src/dymoprint/gui/gui.py

```diff
@@ -20,31 +21,28 @@
 )
 
 from dymoprint.gui.common import crash_msg_box
-from dymoprint.lib.constants import DEFAULT_MARGIN_PX, ICON_DIR
+from dymoprint.lib.constants import ICON_DIR
 from dymoprint.lib.dymo_labeler import (
     DymoLabeler,
     DymoLabelerDetectError,
     DymoLabelerPrintError,
 )
 from dymoprint.lib.logger import configure_logging, set_verbose
 from dymoprint.lib.render_engines import RenderContext
-from dymoprint.lib.utils import px_to_mm, system_run
+from dymoprint.lib.utils import system_run
 
 from .q_dymo_labels_list import QDymoLabelList
 
 LOG = logging.getLogger(__name__)
 
 
 class DymoPrintWindow(QWidget):
-    SUPPORTED_TAPE_SIZE_MM = (19, 12, 9, 6)
-    DEFAULT_TAPE_SIZE_MM_INDEX = 1
-
-    label_bitmap: Optional[Image.Image]
+    print_label_bitmap: Optional[Image.Image]
```

Comment:
I read `print` as a verb, and so this reads like a method name rather than an attribute. Maybe rename to something like `label_bitmap_to_print`?